### ISSUE TITLE
no class found 

### ISSUE THREAD

**Issue by ronea23** [y8hamon/HeroRat#1]
import net.herorat.utils.Utils;

The class Utils is missing in the utils package.

**Comment by ronea23** [comment]
And one more class Base64 is also missing.

**Comment by schlei-media** [comment]
Yes thats very sadly. Please reup your Utils.java and Base64.java

**Comment by ronea23** [comment]
hi what you mean my reup ... only crypto class is found, how to get these two classess ?? i

**Comment by y8hamon** [comment]
Sorry for the delay.
Please find the missing classes here: https://gist.github.com/y8hamon/663d1629b067aba248ea

**Comment by schlei-media** [comment]
Your class net.herorat.utils.Utils is still missing.
Every Image Import isn`t working.
e.g.:
	ImageIcon icon_tab3 = new ImageIcon(Utils.toByteArray(this.getClass().getClassLoader().getResourceAsStream("/images/capture.png")));

I fix it by changing Image imports to:
		ImageIcon icon_tab3 = new ImageIcon(getClass().getResource("/images/capture.png"));
		pane_tabs.addTab("Screen", icon_tab3, panel_tab3, "Remote control your computers");

But i think that`s not the good way ;-)

**Comment by y8hamon** [comment]
In fact the image imports are not working and there is an obvious reason to
that: There is a launcher which download and execute the second jar at
runtime. It's a jar injection.

2015-05-06 14:34 GMT+02:00 schlei-media <notifications@github.com>:

> Your class net.herorat.utils.Utils is still missing.
> Every Image Import isn`t working.
> e.g.:
> ImageIcon icon_tab3 = new
> ImageIcon(Utils.toByteArray(this.getClass().getClassLoader().getResourceAsStream("/images/capture.png")));
>
> I fix it by changing Image imports to:
> ImageIcon icon_tab3 = new
> ImageIcon(getClass().getResource("/images/capture.png"));
> pane_tabs.addTab("Screen", icon_tab3, panel_tab3, "Remote control your
> computers");
>
> But i think that`s not the good way ;-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/y8hamon/HeroRat/issues/1#issuecomment-99432210>.
>



-- 
Yoann HAMON
